ci-operator/config/open-cluster-management/application-ui/open-cluster-management-application-ui-main.yaml

```diff
@@ -32,6 +32,44 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: e2e-cluster
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: application-ui
+    env:
+      CLUSTERPOOL_GROUP_NAME: AppLifecycle
+      CLUSTERPOOL_HOST_NAMESPACE: app
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: ocm-app-ui-clusterpool
+      CLUSTERPOOL_LIST_INCLUSION_FILTER: prow
+    test:
+    - as: test
+      commands: |
+        KUBECONFIG=${SHARED_DIR}/hub-1.kc oc cm attach cluster --name managed --cluster-kubeconfig ${SHARED_DIR}/managed-1.kc \
```

Comment:
Tried to add another test step under test, but got failure when `make update`. Combined as one now
```
time="2021-06-07T14:50:55Z" level=fatal msg="Failed to generate jobs" error="invalid ci-operator config: invalid configuration: 'tests[0].test[0].resources' should have at least one request or limit" source=/go/src/github.com/openshift/release/ci-operator/config subdir= target=/go/src/github.com/openshift/release/ci-operator/jobs
make[1]: *** [jobs] Error 1
make: *** [update] Error 2
```